### DISCUSSION TITLE
GameCube blueRetro having error after connecting

### DISCUSSION THREAD

**Discussion by 90sFavKi** [darthcloud/BlueRetro#489] [Q&A] [answered]
Hello, I bought the RetroRosetta (blue retro) for GameCube and n64 about 2 months, I recently decided to use my GameCube again and one of the issues I found where my l and r triggers were un responsive on the menu screen (works once in gameplay) and my A and B buttons where off (I'm using switch pro controller) so decided to change the configs, I have been able to enter the configs page before so I didn't think I would have a problem, unfortunately It doesn't connect. I first tried to connect on my iPhone using the app Bluefy which worked perfectly before, it will say connected then have an error, I tried on my pc it finally worked but when I went back to the main menu it disconnected and wont connect anymore I have attached the pic of what It shows me 
![487AABC9-49F2-4C82-A16D-BDCA869F0628](https://user-images.githubusercontent.com/67302463/207462342-89c60c60-0049-4278-8a14-2b9dc4a320bc.png)

*UDATE* 
The update fixed eveything on the PC end but iPhone (bluefy) is still having errors,  also the L and R triggers are unresponsive in menus for games, they work once your playing but once you go back to the main menu they are unresponsive again. I tried to remap the buttons to fix this and it still is unresponsive in the menu is there any way to fix this ?

**Comment by darthcloud** [comment, marked as answer]
I might have a fix for Bluefy later.

Your L & R trigger problem is that the menu likely use the end buttons while the game us the analog part

Load the GameCube merged trigger presets with the web config at https://blueretro.io/presets

This gif video show you exactly what to do:
https://github.com/darthcloud/BlueRetro/wiki/BlueRetro-BLE-Web-Config-User-Manual#33---example-preset-load

**Reply by 90sFavKi** [reply]
Hey darth thanks that fixed everything ! Can you have like a thread on all your social media for beginners? Because the information is available but we have no idea where to find it 👍🏽 I know Twitter has the ability to create a thread, and the gifs helped so much more better then just written instructions because it shows us how to do things does blue retro have a discord yet ?

**Reply by darthcloud** [reply]
Discord: https://discord.com/invite/n8KmR6v4em

**Comment by darthcloud** [comment]
Bluefy fix is in v1.8.1 https://darthcloud.itch.io/blueretro

**Reply by 90sFavKi** [reply]
Im having a hard time staying connected to blue retro web, connecting in itself is difficult because it will have an error but when am able to connect it only last for a couple seconds then it disconnects by itself  so I can't update unfortunately, I've tired chrome and Microsoft edge and it's the same issue I'm on pc
